crates/codegen/syntax/src/to_parser_code.rs

```diff
@@ -436,15 +436,15 @@ fn precedence_expression_to_parser_code(
         loop {
             let mut elements: Vec<ParserResult> = Vec::new();
 
-            let result = loop {
```

Comment:
I suggest adding a "patch" changeset with this PR to explain the fix.
I will create a new release `0.7.1` to get it out once it is merged.